testhelper/docker/resource/postgres/postgres.go

```diff
@@ -57,7 +58,8 @@ func Setup(pool *dockertest.Pool, d resource.Cleaner, opts ...func(*Config)) (*R
 			"POSTGRES_DB=" + postgresDefaultDB,
 			"POSTGRES_USER=" + postgresDefaultUser,
 		},
-		Cmd: cmd,
+		Cmd:          cmd,
+		PortBindings: internal.IPv4PortBindings([]string{"5432"}),
 	}, func(hc *dc.HostConfig) {
```

Comment:
Don't we need to set PublishAllPorts to `false`?
```GO
postgresContainer, err := pool.RunWithOptions(&dockertest.RunOptions{
		Repository: "postgres",
		Tag:        c.Tag,
		NetworkID:  c.NetworkID,
		Env: []string{
			"POSTGRES_PASSWORD=" + postgresDefaultPassword,
			"POSTGRES_DB=" + postgresDefaultDB,
			"POSTGRES_USER=" + postgresDefaultUser,
		},
		Cmd:          cmd,
		PortBindings: internal.IPv4PortBindings([]string{"5432"}),
	}, func(hc *dc.HostConfig) {
		hc.ShmSize = c.ShmSize
		hc.OOMKillDisable = c.OOMKillDisable
		hc.Memory = c.Memory
	},
		internal.DefaultHostConfig,
	)
```

Reply:
Do we need to set PublishAllPorts to false in `sshserver.go` and `transformer.go` as well?

Reply:
We should set PublishAllPorts to false in all the resources and expose ports only via port binding.